arviz/plots/compareplot.py

```diff
@@ -91,7 +91,7 @@ def plot_compare(
 
     yticks_labels = [""] * len(yticks_pos)
 
-    _information_criterion = ["waic", "loo"]
+    _information_criterion = ["waic", "loo", "WAIC", "LOO"]

```

Comment:
We probably should just the comparison case insensitive, especially because I think this might break arviz now

Reply:
Realized this wont break arviz but still feel like making the whole comparison case insensitive is the way to go